docs/conf.py

```diff
@@ -15,20 +15,20 @@
 import os
 import sys
 
-sys.path.append(
-    os.path.join(os.path.dirname(__file__), '..', '..', 'src')
-)
+# sys.path.append(os.path.join(os.path.dirname(__file__), "..", "src"))
+#
+# import fixate
```

Comment:
So, because the tox setup somehow magically knows how to install fixate we don't need import (satisfying option 1 [here](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html)) . It doesn't work though if one was to use the Makefile, assuming that the primary way to build the documentation will be via tox, do we need to keep the Makefile? Or the make.bat file?

Is there any reason to leave the unused import here?

Reply:
The answer to both you questions is that I was lazily changing things trying to fix a few unrelated issues.

But I think I've made it all better now... watch for an incoming PR

Reply:
@jcollins1983 I've just been doing a `sphinx-build . _build` anyway. I think I noted it elsewhere, but the disadvantage I've found with `tox -e docs` is that the warnings end up being:

```
/Users/josh/Ampcontrol/repos/pyfixate/.tox/docs/lib/python3.12/site-packages/fixate/drivers/funcgen/keysight_33500b.py:docstring of fixate.drivers.funcgen.keysight_33500b.Keysight33500B.get_identity:3: WARNING: Definition list ends without a blank line; unexpected unindent.
```
instead of
```
/Users/josh/Ampcontrol/repos/pyfixate/src/fixate/drivers/funcgen/keysight_33500b.py:docstring of fixate.drivers.funcgen.keysight_33500b.Keysight33500B.get_identity:3: WARNING: Definition list ends without a blank line; unexpected unindent.
```
where in the tox version it's too tempting to click on the source file to change it... but that's the copy tox has made to build.

@clint-lawrence, if tox doesn't install the source or whatever magic you've done, presumably this problem goes away too?

Reply:
To answer my own question... yes that PR fixed it...

```
/Users/josh/Ampcontrol/repos/pyfixate/docs/../src/fixate/drivers/funcgen/keysight_33500b.py:docstring of fixate.drivers.funcgen.keysight_33500b.Keysight33500B.get_identity:3: WARNING: Definition list ends without a blank line; unexpected unindent.
```
works!

Reply:
The `import fixate` is to be able to set the `version` and `release` attributes. They were temporarily commented out while I was trying to fix a dependency problem.

Manipulating the python path allows building the docs over the source tree, instead of the installed python files in a virtual env.